docfx_project/articles/bootstrapping-trust/part-05.bootstrapping-trust.md

```diff
@@ -13,56 +13,58 @@ have not, please consider reading the whole series:
 
 ### Ziti
 
-In this series of articles we are exploring bootstrapping trust and what
-that means and how it enables Zero Trust security methodologies. Ziti
+In this series of articles we are exploring bootstrapping trust, what
+that means, and how it enables Zero Trust security methodologies. Ziti
 provides a method to bootstrap trust via its enrollment process. For
-Ziti the enrollment process is bootstrapping trust. All identities in
-Ziti have a key pair that identifies that individual. The enrollment
-process abstracts away the process of setting up keys, certificates,
-CSRs, CAs, and having each software component of Ziti trust the
-necessary certificates. The Ziti SDK can be embedded within any
-application and enroll with a Ziti network - instantly granting it
-secure, encrypted, and trusted connections that provide part of Ziti's
-Zero Trust model.
-
-
-Ziti has the concept of the Edge, which sits on top of the Fabric. The
-Fabric provides long haul mesh routing while the Edge focuses on
-enrolling Ziti components, managing access via policies, and maintaining
-the trust necessary to provide the foundation of a Zero Trust network
-without the hassle of setting it up yourself. Together they are a
-powerful combo of optimized long halt routing and access and trust
+Ziti the enrollment process is bootstrapping trust. This trust must be
+in place as all connections in Ziti require verification. All identities
+in Ziti have a key pair that identifies that individual. The enrollment
+process abstracts the steps of setting up keys, certificates, CSRs, CAs,
+and deploying them to the proper locations. In addition, the Ziti SDK
```

Comment:
nit: "the Ziti SDK" --> a Ziti SDK or the suite of Ziti SDKs, or any of the Ziti SDKs or .. anything that implies there's more than one sdk